style.css

```diff
@@ -0,0 +1,31 @@
+.image{
+    float:left;
+    margin: 44px 70px 7px 0;
+}
+.div-flex{
+    display: flex;
+    flex-direction: column;
+   border-bottom: 1px solid ;
```

Comment:
Should add empty space at the beginning of the line. Also should delete empty space before ';'